### ISSUE TITLE
Bug: Accessibility: 508: Keyboard Access issue

### ISSUE THREAD

**Issue by binwang89** [18F/dol-whd-14c#627]
Remediate accessibility issues in below pages, according to acceptance criteria noted in #198.
 
Bug description:
1. An interactive element or function cannot be accessed or activated by keyboard. 
1.1 Example: https://dol-whd-section14c-qa.aisdemos.com/#!/dashboard  - "clear all responses" function brings up a popup screen, it is not available for keyboard users.
1.2 Work Sites & Employees: Employee data (in the table) can not be sorted by keyboard.

2. At any time, there is no visual indication of the current focus (loss of focus). 
Example: Work Sites & Employees:  The focus is lost after last "Delete" button and before "Back" button in the table.

3. The visual focus does not move to revealed content and no description of the content change is provided . 
3.1 Example: https://dol-whd-section14c-qa.aisdemos.com/#!/login
Create Account - When the user enters weak password, the screen provides instructions, however the focus does not fall on it, thus the screen reader does not read it out.
3.2 Log in - if wrong password or username is entered focus should fall on the error message.
https://dol-whd-section14c-qa.aisdemos.com/#!/changePassword  - focus should fall on error message.

4. The tab order is not logical. 
4.1 Example: https://dol-whd-section14c-qa.aisdemos.com/#!/changePassword  - once the password is successfully changed, tabbing of the screen is not available.
4.2 https://dol-whd-section14c-qa.aisdemos.com/#!/login - if you enter incorrect email address for registration, click Register, next tabbing does not start from the error message but takes you to the footer links

Acceptance Criteria
Functionality: No functional changes. 
Accessibility:  
1.	Keyboard Accessibility
[ ] 1.1 An interactive element or function can be accessed or activated by keyboard
[ ] 1.2 keyboard "trap" not found
[ ] 1.3 if non-standard keyboard commands are needed and they are documented
[ ] 1.4 TITLE provides information and equivalent information is found through text or visual context
[ ] 1.5 The visual focus can be determined at all times.
[ ] 1.6 focus always appears on the element it is programmatically on.
[ ] 1.7 if focus remains in the modal dialog box until the box is closed.
[ ] 1.8 if focus moves to revealed content OR a description of the content change is provided.
[ ] 1.9 the tab order is logical

**Comment by EStriegel** [comment]
#397 is complete.  This issue can be addressed.

**Comment by binwang89** [comment]
close this user story, put the remaining item 2 to new user story #772

**Comment by NathayaMDOL** [comment]
All issues have been fixed (except #2) and pushed to Dev.
Branch "whd-aqa-627-nmankong-508-keyboard-access-issue"

**Comment by binwang89** [comment]
#776 will replace item 2.   we can close this user story.

**Comment by binwang89** [comment]
@NathayaMDOL 
1.	An interactive element or function cannot be accessed or activated by keyboard.
1.1 Example: https://dol-whd-section14c-qa.aisdemos.com/#!/dashboard - "clear all responses" function brings up a popup screen, it is not available for keyboard users.
1.2 Work Sites & Employees: Employee data (in the table) can not be sorted by keyboard.
**Worked in Google Chrome, however, it is still not working in IE.** 

2.	At any time, there is no visual indication of the current focus (loss of focus).
Example: Work Sites & Employees: The focus is lost after last "Delete" button and before "Back" button in the table.
**Worked in Google Chrome, however, it is still not working in IE**

3.	The visual focus does not move to revealed content and no description of the content change is provided .
3.1 Example: https://dol-whd-section14c-qa.aisdemos.com/#!/login
Create Account - When the user enters weak password, the screen provides instructions, however the focus does not fall on it, thus the screen reader does not read it out.
**Not working in chrome/IE**

3.2 Log in - if wrong password or username is entered focus should fall on the error message.
https://dol-whd-section14c-qa.aisdemos.com/#!/changePassword - focus should fall on error message.
**Working in Both chrome/IE**

4.	The tab order is not logical.
4.1 Example: https://dol-whd-section14c-qa.aisdemos.com/#!/changePassword - once the password is successfully changed, tabbing of the screen is not available.
**Working in Both chrome/IE**

4.2 https://dol-whd-section14c-qa.aisdemos.com/#!/login - if you enter incorrect email address for registration, click Register, next tabbing does not start from the error message but takes you to the footer links
**Working in Both chrome/IE**

@NathayaMDOL Please see attached the testing results
[#627 testing results.docx](https://github.com/18F/dol-whd-14c/files/2307647/627.testing.results.docx)

**Comment by binwang89** [comment]
@NathayaMDOL I created #789 for remaining items. I will close this one for now.
